packages/shared/src/utils/contract/contract.ts

```diff
@@ -32,13 +32,19 @@ export const isValidToSell = (contract_info: TContractInfo) =>
 
 export const hasContractEntered = (contract_info: TContractInfo) => !!contract_info.entry_spot;
 
-export const isAccumulatorContract = (contract_type: string) => /ACCU/i.test(contract_type);
+export const hasTwoBarriers = (contract_type = '') => /EXPIRY|RANGE|UPORDOWN/i.test(contract_type);
 
-export const isMultiplierContract = (contract_type: string) => /MULT/i.test(contract_type);
+export const isAccumulatorContract = (contract_type = '') => /ACCU/i.test(contract_type);
 
-export const isVanillaContract = (contract_type: string) => /VANILLA/i.test(contract_type);
+export const isMultiplierContract = (contract_type = '') => /MULT/i.test(contract_type);
 
-export const isCryptoContract = (underlying: string) => /^cry/.test(underlying);
+export const isVanillaContract = (contract_type = '') => /VANILLA/i.test(contract_type);
+
+export const isSmartTraderContract = (contract_type = '') => /RUN|EXPIRY|RANGE|UPORDOWN|ASIA/i.test(contract_type);
```

Comment:
@kate-deriv does it always include ASIAN? if yes, it'd better to put ASIAN instead of ASIA☺️

Reply:
@maryia-deriv it include ASIAN: ASIANU and ASIAND. I'll put ASIAN